meta-triggerfish/recipes-core/base-files/base-files/fstab

```diff
@@ -0,0 +1,5 @@
+/dev/root            /                    auto       defaults              1  1
```

Comment:
This looks like the default `fstab` (https://github.com/openembedded/openembedded-core/blob/kirkstone/meta/recipes-core/base-files/base-files/fstab) since you aren't mounting the `persist` partition. Can you remove this and check if everything is still ok.